event-handlers/src/test/resources/evaluator/candidate_verifiedCreator_with_nonNviInstitution.json

```diff
@@ -244,7 +244,7 @@
             "verificationStatus": "Verified"
           },
           "role": {
-            "type": "Creator"
+            "type": "RoleOther"
```

Comment:
This file had the same contributor duplicated with different roles, which I believe is invalid. Changing the roles to match here.